west.yml

```diff
@@ -184,7 +184,7 @@ manifest:
         - hal
     - name: hal_nordic
       revision: 64bd075a06f6a16d6ea9101cda82b41465a35211
-      path: modules/hal/nordic
+      path: pull/160/head
```

Comment:
update with sha after https://github.com/zephyrproject-rtos/hal_nordic/pull/160 is merged

Reply:
Please update SHA (and rebase): `4362d0fcdbac80942212bffc999654beac7ab30e`

Reply:
Done.

I also fixed a simple mistake that somehow went unnoticed so far: originally I changed `path` instead of `revision` field :) It's corrected now.